src/Features/header/components/Cities/Cities.tsx

```diff
@@ -1,39 +1,46 @@
 import { FiNavigation } from 'react-icons/fi';
 import { ReactElement, useMemo, useCallback } from 'react';
 
-export const cityLocations = [
-  { city: 'Москва', branch: 92 },
-  { city: 'Санкт-Петербург', branch: 16 },
-  { city: 'Калининград', branch: 10 },
-  { city: 'Красногорск', branch: 5 },
-  { city: 'Воронеж', branch: 4 },
-  { city: 'Долгопрудный', branch: 3 },
-  { city: 'Красноярск', branch: 3 },
-  { city: 'Раменское', branch: 3 },
-];
+import { useFetchAllCityLocationsQuery } from '../Sidebar/SidebarSlice';
+import { LoadingSpinner } from './LoadingSpinner';
+
+interface CityList {
```

Comment:
create Cities.types.ts file and put type there. The component file can contain 2 things: the prop types for this component and the component itself